crates/db_views/src/combined/modlog_combined_view.rs

```diff
@@ -78,101 +88,117 @@ impl ModlogCombinedViewInternal {
     // The query for the admin / mod person
     // It needs an OR condition to every mod table
     // After this you can use person::id to refer to the moderator
-    let moderator_names_join = show_mod_names_expr
-      .or(person::id.nullable().eq(mod_person_id))
-      .and(
-        admin_allow_instance::admin_person_id
-          .eq(person::id)
-          .or(admin_block_instance::admin_person_id.eq(person::id))
-          .or(admin_purge_comment::admin_person_id.eq(person::id))
-          .or(admin_purge_community::admin_person_id.eq(person::id))
-          .or(admin_purge_person::admin_person_id.eq(person::id))
-          .or(admin_purge_post::admin_person_id.eq(person::id))
-          .or(mod_add::mod_person_id.eq(person::id))
-          .or(mod_add_community::mod_person_id.eq(person::id))
-          .or(mod_ban::mod_person_id.eq(person::id))
-          .or(mod_ban_from_community::mod_person_id.eq(person::id))
-          .or(mod_feature_post::mod_person_id.eq(person::id))
-          .or(mod_hide_community::mod_person_id.eq(person::id))
-          .or(mod_lock_post::mod_person_id.eq(person::id))
-          .or(mod_remove_comment::mod_person_id.eq(person::id))
-          .or(mod_remove_community::mod_person_id.eq(person::id))
-          .or(mod_remove_post::mod_person_id.eq(person::id))
-          .or(mod_transfer_community::mod_person_id.eq(person::id)),
-      );
+    let moderator_names_join = person::table.on(
```

Comment:
These are actually boilerplate, I just moved the `table.on` into the join variable.